src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -1060,6 +1060,38 @@ actionWatermark
     );
     ```
     
+actionGetAdminOrderButtons
```

Comment:
This one isn't new in 1.7.7?

Reply:
Yes it is The `available since` part is missing indeed, I'm gonna add it